src/utils/bwellPersonFinder.js

```diff
@@ -0,0 +1,77 @@
+const {assertTypeEquals} = require('./assertType');
+const {DatabaseQueryFactory} = require('../dataLayer/databaseQueryFactory');
+
+class BwellPersonFinder {
+    /**
+     * constructor
+     * @param {DatabaseQueryFactory} databaseQueryFactory
+     */
+    constructor(
+        {
+            databaseQueryFactory
+        }
+    ) {
+        /**
+         * @type {DatabaseQueryFactory}
+         */
+        this.databaseQueryFactory = databaseQueryFactory;
+        assertTypeEquals(databaseQueryFactory, DatabaseQueryFactory);
+    }
+
+    /**
+     * finds the bwell person ID associated with a provided patient ID
+     * @param {string} patientId
+     * @return {Promise<string>}
+     */
+    async getBwellPersonIdAsync({ patientId}) {
+        const databaseQueryManager = this.databaseQueryFactory.createQuery({
+            resourceType: 'Person',
+            base_version: '4_0_0'
```

Comment:
It feels like I should be able to get this 4_0_0 from somewhere instead of having it hard-coded. But the other examples I found were basically just hardcoding. If there's a better way to get the value, please let me know.